src/base/image_reader.h

```diff
@@ -44,9 +44,18 @@ struct ImageReaderOptions {
   // Path to database in which to store the extracted data.
   std::string database_path = "";
 
-  // Root path to folder which contains the image.
+  // Root path to folder which contains the images.
   std::string image_path = "";
 
+  // Optional root path to folder which contains image masks. For a given image,
```

Comment:
It would be great, if you could copy over this to the documentation to docs/faq.rst as well. I am sure this will be a feature that quite a few people will find useful :-)